torch/distributed/_tensor/ops/matrix_ops.py

```diff
@@ -327,3 +357,19 @@ def scaled_dot_product_attention_backward_strategy(
             all_strategies.append(strat)
 
     return OpStrategy(all_strategies)
+
+
+@register_op_strategy(aten.constant_pad_nd.default)
+def constant_pad_nd_strategy(mesh: DeviceMesh, op_schema: OpSchema) -> OpStrategy:
```

Comment:
nit: maybe we should add a TODO here as for constant pad, I think we can "following the sharding" if the padding dimension is not the sharding dimension

Reply:
added TODO